docker/base-buster.Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM debian:buster
```

Comment:
can we use a build arg rather than cutting and pasting the entire dockerfile?

Reply:
compare: https://github.com/matrix-org/sytest/pull/745

Reply:
yeah that works better, done